example_test.go

```diff
@@ -284,3 +284,26 @@ func ExampleEmulate() {
 		panic(err)
 	}
 }
+
+func ExamplePrintToPDF() {
+	ctx, cancel := chromedp.NewContext(context.Background(), chromedp.WithDebugf(log.Printf))
```

Comment:
remove the Debugf part? it might trick users into thinking that it's necessary to get the example working, while it isn't.